Makefile

```diff
@@ -0,0 +1,8 @@
+NAME      := MACSio
+SRC_EXT   := gz
+
+include packaging/Makefile_packaging.mk
+
+test:
+	$(call install_repos,$(NAME)@$(BRANCH_NAME):$(BUILD_NUMBER))
+	yum -y install $(NAME)
```

Comment:
I guess this seems like it could/should be the minimum/default test we could use in `packaging/` for RPM builds when a repo doesn't define something more extensive.  Feel free to add that to [daos-stack/packaging](https://github.com/daos-stack/packaging) if you like.  :wink: